dashbuilder/dashbuilder-shared/dashbuilder-services-api/src/main/java/org/dashbuilder/transfer/DataTransferServices.java

```diff
@@ -27,12 +27,16 @@
     public static final String EXPORT_FILE_NAME = "export.zip";
     public static final String IMPORT_FILE_NAME = "import.zip";
     public static final String COMPONENTS_EXPORT_PATH = "dashbuilder/components/";
-
+    
+    public static final String EXPORT_LOCATION_PROP = "dashbuilder.import.base.dir";
```

Comment:
Is it safe to define the property like this? The `dashbuilder.import.base.dir` is defined also in [another place](https://github.com/kiegroup/appformer/blob/3d7730d8e16ac4d51459ba72162935bba61c8795/dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/RuntimeOptions.java#L47) for dashbuilder runtime and it might happen that the property will change only in one of the places and they will not be the same (although the intention is to have the same String for the two properties)

Reply:
That's a good point! I did forget to change this property name, but I will change it and it will make easier to distinguish from the Runtime one. Thanks!